src/main/xsl/preprocess/maprefImpl.xsl

```diff
@@ -389,6 +394,23 @@ See the accompanying LICENSE file for applicable license.
     </xsl:call-template>
   </xsl:template>
 
+  <xsl:template match="@*" mode="preserve-submap-attributes">
+    <xsl:attribute name="dita-ot:submap-{local-name()}" select="."/>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class,' topic/title ')]" mode="preserve-submap-title-and-topicmeta">
+    <dita-ot:submap-title class="+ topic/foreign ditaot-d/submap-title ">
```

Comment:
Couple of reasons:
1. Main reason: `<navtitle>` is not allowed as a direct child of `topicref`. I could easily picture some step failing when a fallthrough rule intended for normal `navtitle` processing fires based on that element, when it should just be processing `topicref` elements.
1. If we're going to use / allow elements that would be out-of-context against the original grammar, I'd prefer to just stick with `topic/title` -- use the exact match rather than a close match. (Or ... were you suggesting that we actually create `<topicmeta><navtitle>Title here</></>` inside the `submap` element? I could see that... except based on the submap `@class` value, I bet the current `mappull` would then warn "you have a topic group with a title and that's wrong")